src/character/mod.rs

```diff
@@ -8,6 +8,13 @@ pub mod class;
 use crate::randomizer::{random, Randomizer};
 use class::Class;
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone, Copy)]
+pub enum Condition {
```

Comment:
consider calling this StatusEffect, which is [the convention](https://en.wikipedia.org/wiki/Status_effect)

Reply:
I was looking for that. Thanks!